14/umbraco-deploy/getting-started/deploy-settings.md

```diff
@@ -56,7 +56,6 @@ For illustration purposes, the following structure represents the full set of op
             "TransferDictionaryAsContent": false,
             "IgnoreMissingLanguagesForDictionaryItems": false,
             "SetEmptyDictionaryItemsOnTransfer": true,
-            "IgnoreMissingLanguagesForDictionaryItems": false,
```

Comment:
In your description, you do not mention removing this? 🤔 
Perhaps it's an error?

Reply:
Must be a leftover from a previous commit, I'll remove asap